### ISSUE TITLE
Installation

### ISSUE THREAD

**Issue by krober10nd** [dengwirda/inpoly-python#3]
Just making a new issue for easy searching in the future. 

For installation,  I had to export the include path for numpy headers in order for the program to build with cython correctly. 

Like this:

```
python
import numpy 
numpy.get_include()
```

and then I copied that last path and put it here:
```
export CFLAGS="-I /Users/Keith/firedrake/lib/python3.7/site-packages/numpy/core/include $CFLAGS"
```

After that things appear to work and are registering 150x speed ups over `matlplotlib.path.contains` 🥇

**Comment by krober10nd** [comment]
Yep. Everything is working great with the installation on my mac. I ran all the instructions on the README and also tried via pip and didn't need to export any flags to the LD path. 

My auto formatter is just telling me that the variable `ymin` is unused. 

The other small thing is should `ftol` and `lbar` be declared as `double` in the function signature? 

Otherwise, everything looks great! If you're really pushing the limits for a big problem it may be possible to use thread based parallelism using `prange` instead of `range`.

https://cython.readthedocs.io/en/latest/src/userguide/parallelism.html

**Comment by krober10nd** [comment]
Did you think about patenting this algorithm? The `numpy` community may have interests in building this algorithm into their default library since it appears there is no existing algorithm with better complexity bounds.

**Comment by dengwirda** [comment]
Alright --- I think there's now the full workflow up and running: [`inpoly` is on `PyPi` with both `sdist` and `bdist_wheel` files](https://pypi.org/project/inpoly/#files) available for `Python 3.5 => 3.8` on `linux`, `macOS` and `windows`. If you do `pip install inpoly` you should now get the pre-built binaries installed automagically, a'la `numpy`, etc...

As you saw earlier, I've used `travis` + `cibuildwheel` to build all of the binaries and upload them to `PyPi`. This process involved many tears; but ultimately (I guess) tears of joy! I think all should be in place now to maintain going forward --- `travis` will run on any commit to the `inpoly-python` repo, but will only upload new `bdist_wheel`'s to `PyPi` for tagged releases (that build successfully). I should probably add unit tests for `travis` to run too, but that can happen another day.

I'll reopen #1 to discuss a few of the other performance-type things.

**Comment by krober10nd** [comment]
Wow that's awesome! I have never seen the CI system used for that purpose but it makes sense. I guess there was some boiler plate code to figure out for the .ci/config file (there almost always is).
